windows-installer/setup.nsi

```diff
@@ -220,6 +230,15 @@ Section "Install"
     ${If} $4 == "1"
       StrCpy $2 "  pos_file '${MAIN_INSTDIR}\pos\winevtlog.pos'$\r$\n"
     ${EndIf}
+	
+    ; Count the number of 'VERSION_LABEL_PLACE_HOLDER' instances.  It should only
+    ; ever be 0 or 1.  We only have it in the template file once.
+    ${WordFind} "$3" "VERSION_LABEL_PLACE_HOLDER" "#" $4
+
+    ; If we hit the place holder line replace it with the proper label.
+    ${If} $4 == "1"
+      StrCpy $2 "    $\"agent$\":$\"windows-stackdriver-logging-agent ${VERSION}$\"$\r$\n"
```

Comment:
Do we really want to tag **every** log message with the agent version? Could we look at other mechanisms, e.g., write the version to the event log once on startup?